app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
```

Comment:
- [ ] Since the User has_many of these attributes through the `author_id`, we need to specify that here also.

  ```
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
  ```